hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieCompactionConfig.java

```diff
@@ -260,6 +264,11 @@ public Builder withCleanBootstrapBaseFileEnabled(Boolean cleanBootstrapSourceFil
       return this;
     }
 
+    public Builder withCleanFailedWritesPolicy(HoodieFailedWritesCleaningPolicy failedWritesPolicy) {
```

Comment:
withFailedWritesCleaningPolicy(). can we make this consistent everywhere.

Reply:
Done, is there an easy way to generate a with... for these variables ? I'm manually typing them hence end up introducing errors.